docs/catalyst_101.md

```diff
@@ -136,6 +136,159 @@ Once the existing members have voted to offer you membership, it’s up to you t
+  - discuss status and whether we should roll into the new sprint; tag with new sprint if necessary.
+- for items in backlog:
+  - discuss whether we should pull into the new sprint; tag with new sprint if necessary.
+- ad-hoc discussion topics
+
+Mid sprint checkin (Monday, week 2):
+
+- for each item in the sprint:
+  - discuss status
+- ad-hoc discussion topics
+
+Sprint review (Thursday, week 2):
+
+- for each item in the sprint:
+  - discuss status
+- ad-hoc discussion topics
+
+Outside of sprint meetings:
+
+- Dazhong ingests new issues and PRs to the GH Projects board manually (don't worry, this takes like 5 minutes)
+- Everyone moves issues / linked PRs within the GH project when status changes
+
+### Current experiments
+
+For each experiment:
+
+- what's the goal?
+- what's the actual thing we do?
+- how will we assess success?
+- how do we know that sticking to it will feel better than giving up?
+
+#### Only pull scoped, prioritized tickets into the sprint
+
+Goal: make it easier to decide what to work on by agreeing on priority as a team and providing sufficient context for team members to get started on each item.
+
+Activity:
+
+Use the mid sprint checkin as a parallel work block to pull items into Backlog - aiming for at least 2N items in Backlog, where N is the number of people on the team. All items in Backlog should have scope and a next step:
+
+- scope: when would we consider this issue to be done? what criteria have to be met, or what behavior has to exist?
+- next step: what's the first step to getting this working?
+
+In the sprint review, instead of discussing status for each item in the sprint, prioritize the tickets that are in Backlog according to the following schema:
+
+- "Urgent" -> blocking work on high-priority tickets
+- "High" -> if we do not finish this during the next two weeks, we will not hit our quarterly goals
+- "Medium" -> we'd like to finish this during the next two weeks
+- "Low" -> if there's not other stuff to do, it's worth knocking out
+
+In the sprint planning:
+
+- Pull high priority items from in-flight and backlog into the sprint until it feels like we have enough, but not too much, work.
+  - If a low priority item is in flight, we should bump it for higher-priority work.
+
+Outside of the meetings:
+
+- If a new issue comes up, it goes in the Icebox unless there's a compelling case for it to be Urgent, in which case we talk about it as a team and pull it into the sprint if it's truly Urgent.
+
+Success criteria:
+
+- it feels easier than before to start work on a new ticket when your old ticket is done
+- minimal time spent doing homework outside of meetings
+
+Accountability/how we'll stick to it:
+
+- parallel work blocks are more engaging than "everyone talk through everything in series."
+
+#### Limit work-in-progress
+
+Goal: reduce overwhelm from "too many things going on at once" and identify bottlenecks in our process
```

Comment:
On the flipside of too manything... sometimes I feel like there's nothing currently in a sprint that I have the necessary context to work on productively, and this can result in feeling like there's nothing I'm really allowed to work on. So either I have to sneak around and work on something I do know about without it being in the sprint, or work inefficiently on something I don't know enough about to actually be helpful.

Reply:
I think there are two ways we can take this:

* pull more stuff into the sprint when we run out of stuff within one person's context
* use this as a forcing function to share context between people

And I think this might be a bit of a case-by-case situation - i.e., is there other high priority stuff that one could knock out? (e.g. sloan stuff -> archiver)

Maybe this goes into "Help!" in the checkins - "Help! I don't know what to work on next."